metrics/filesystem.go

```diff
@@ -14,38 +14,28 @@ type FilesystemGenerator struct {
 	IgnoreRegexp *regexp.Regexp
 }
 
-var dfColumnSpecs = []util.DfColumnSpec{
-	{Name: "size", IsInt: true},
-	{Name: "used", IsInt: true},
-}
-
 var sanitizerReg = regexp.MustCompile(`[^A-Za-z0-9_-]`)
 
 // Generate the metrics of filesystems
 func (g *FilesystemGenerator) Generate() (Values, error) {
```

Comment:
I found that tests for `Generate()` is so simple 😱
https://github.com/mackerelio/mackerel-agent/blob/master/metrics/filesystem_test.go